etc/spgw_u.conf

```diff
@@ -20,7 +20,8 @@
 ################################################################################
 SPGW-U =
 {
-    FQDN = "gw@GW_ID@.spgw.node.epc.mnc@MNC03@.mcc@MCC@.@REALM@";
+    #FQDN = "gw@GW_ID@.spgw.node.epc.mnc@MNC03@.mcc@MCC@.@REALM@";
```

Comment:
You can remove the commented line.

Or better you can explain both formats as comment (the 4G case and the 5G case)